### PR TITLE
Add installation instructions for New Relic in PHP 5 and 7 examples

### PR DESCRIPTION
**Pull request by aramalipoor**

### CODE REVIEW

File: php/5/supervisord.conf [hunk 1]
```diff
@@ -21,3 +21,11 @@ stdout_logfile=/dev/stdout
 stderr_logfile=/dev/stderr
 stdout_logfile_maxbytes=0
 stderr_logfile_maxbytes=0
+
+; Output Newrelic logs to stdout so they are visible in Pod's log
+[program:newrelic]
+command=tail -f /var/log/newrelic/*
```
Comment (aramalipoor) [inline php/5/supervisord.conf]: Should we keep this?

**Comment by alikhajeh1** [comment]
@aramalipoor this is great, most users won't use or need NewRelic though so I think it's better if this is a separate example code snippet that users can add to their Dockerfile/supervisor files.

File: php/5/supervisord.conf (hunk 1 above)
Comment (alikhajeh1) [reply php/5/supervisord.conf]: How noisy is the logging? if it's just at boot then it's useful for debugging if their Dockerfile changes didn't work.

**Comment by alikhajeh1** [comment]
@aramalipoor does this work for non-alpine image aye? seems like it should

**Comment by aramalipoor** [comment]
@alikhajeh1 not so much different execpt the Newrelic version it's downloading which `linux-musl`, meaning ready for alpine not debian.

File: php/5/supervisord.conf (hunk 1 above)
Comment (aramalipoor) [reply php/5/supervisord.conf]: Looks like only logs when daemon creates a connection to Newrelic servers, shouldn't be that noisy.

**Comment by alikhajeh1** [comment]
1. Sounds good, I thought you wanted to keep code snippets in this repo, so you can reference them in docs using docco? 
2. Can the Dockerfile comments mention this and what non-alpine users need to do?

**Comment by aramalipoor** [comment]
@alikhajeh1 Now I have kept the snippets in `/extra` directory.

**Pull request merged**